### ISSUE TITLE
Feature request: Toggle checking for updates

### ISSUE THREAD

**Issue by 3nly** [Slion/Fulguris#88]
Hello, I appreciate the work you're doing on this browser, it's fantastic. I'm a recent user of it after Lightning browser seems to have become abandonware... or least has painfully slow releases.

One thing bothers me however,  every time the browser gets launched it connects to `slions.net ` to check for updates and there is no way to disable that. A toggle to enable auto-updates would be nice so the browser stays silent on launch.

Thanks again for the hard work.

**Comment by Slion** [comment]
> toggle to enable auto-updates would be nice

Makes sense, I could do that. In the meantime you could use either Google Play or F-Droid variants which do not check for update.

 > Thanks again for the hard work.

Thanks for using Fulguris and taking the time to come here and contribute.

2026-04: I wonder if we also get a crash if slions.net is offline
